commands/config/default-client-config.yml

```diff
@@ -46,3 +46,5 @@ providers:
       - http://localhost:3000/login-callback
       - http://localhost:10001/login-callback
       - http://localhost:11110/login-callback
+
+send_access_token: false
```

Comment:
Would there be a benefit in having this as a per provider option instead? I understand that this is mainly motivated by Google not providing enough information, so we could be specific and only send this for providers where it is necessary, e.g., if multiple providers are used.

Reply:
You are right, this should be scoped to the provider